developers/blobstream-contracts.md

```diff
@@ -109,7 +109,7 @@ on Ethereum to query the `BlobstreamX.sol` contract for relayed `DataRootTuple`s
 The single interface method `verifyAttestation` verifies a Merkle inclusion
 proof that a `DataRootTuple` is included under a specific batch (indexed by
 batch nonce). In other words, analogously it verifies that a specific block
-header is included in the canonical Celestia chain.
```

Comment:
Wait you should keep this, since it's about the canonical Celestia chain, not the canonical Blobstream contract, no?

Reply:
There is only one Celestia chain, so why does it matter anyway, you don't have to refer to it as canonical

Reply:
Agreed, canonical gives the impression of having multiple Celestia chains

Reply:
aren't there multiple "Celestia" chains, which are `celestia` a.k.a. Mainnet Beta, `mocha-4`, and `arabica-11`?

Reply:
should we resolve this and move forward with original change?

Reply:
Hmmm I guess it's true there's only one Celestia chain.